src/pudl/analysis/state_demand.py

```diff
@@ -433,7 +434,7 @@ def melt_ferc714_hourly_demand_matrix(
 
 
 @asset(
-    compute_kind="Python",
+    compute_kind="pandas",
```

Comment:
I started adding these `compute_kind` tags to the `out` assets when I thought we were going to have some SQL views mixed in. I don't know how we want to use them going forward, given 99% of our assets use pandas.